packages/wrapped-keys/src/lib/api/export-private-key.ts

```diff
@@ -0,0 +1,55 @@
+import { decryptToString } from '@lit-protocol/encryption';
+
+import { CHAIN_ETHEREUM, LIT_PREFIX } from '../constants';
+import { fetchPrivateKeyMetadata } from '../service-client';
+import { ExportPrivateKeyParams, ExportPrivateKeyResult } from '../types';
+import {
+  getFirstSessionSig,
+  getPkpAccessControlCondition,
+  getPkpAddressFromSessionSig,
+} from '../utils';
+
+/** Exports a previously persisted private key from the wrapped keys service for direct use by the caller, along with the keys metadata
+ *
+ * @param { ExportPrivateKeyParams } params Parameters required to export the private key
+ */
+export async function exportPrivateKey(
+  params: ExportPrivateKeyParams
+): Promise<ExportPrivateKeyResult> {
+  const { pkpSessionSigs, litNodeClient } = params;
+
+  const sessionSig = getFirstSessionSig(pkpSessionSigs);
+  const pkpAddress = getPkpAddressFromSessionSig(sessionSig);
+  const allowPkpAddressToDecrypt = getPkpAccessControlCondition(pkpAddress);
+
+  const privateKeyMetadata = await fetchPrivateKeyMetadata({
+    sessionSig,
+    litNetwork: litNodeClient.config.litNetwork,
+  });
+
+  const { ciphertext, dataToEncryptHash, ...privateKeyMetadataMinusEncrypted } =
+    privateKeyMetadata;
+
+  const decryptedPrivateKey = await decryptToString(
+    {
+      accessControlConditions: [allowPkpAddressToDecrypt],
+      chain: CHAIN_ETHEREUM, // FIXME: This won't always be true.
```

Comment:
I think it will as it's just the PKP Eth Address?

Reply:
:+1: removed FIXME